properties.schema

```diff
@@ -238,6 +238,15 @@
                       "help": "What exit status to use if the course is complete."
                     }
                   }
+                },
+                "_showCookieLmsResetButton": {
```

Comment:
There are two types of schema now can you make sure to update the ones in the schema folder as well please? They're a slightly different format, but they're for the new AAT

Reply:
That one keeps catching me out! Thanks